### PR TITLE
[4] Non-Controversial CodeStyle Cleanup for /installation

### PR DESCRIPTION
**Pull request by PhilETaylor**
[4] Non-Controversial CodeStyle Cleanup for /installation

Code review

### CODE REVIEW

File: installation/src/Controller/InstallationController.php [hunk 1]
```diff
@@ -145,13 +145,13 @@ public function populate()
...
 			'custom1' => 'localise',
-			'custom2' => 'custom'
+			'custom2' => 'custom',
 		];
 
 		$schema = $files[$step];
 		$serverType = $db->getServerType();
 
-		if (in_array($step, ['custom1', 'custom2']) && !is_file('sql/' . $serverType . '/' . $schema . '.sql'))
+		if (\in_array($step, ['custom1', 'custom2']) && !is_file('sql/' . $serverType . '/' . $schema . '.sql'))
```
Comment (HLeithner) [inline installation/src/Controller/InstallationController.php:154]:
```suggestion
		if (\in_array($step, ['custom1', 'custom2']) && !\is_file('sql/' . $serverType . '/' . $schema . '.sql'))
```

File: installation/src/Form/Rule/UsernameRule.php [hunk 2]
```diff
@@ -39,9 +39,9 @@ public function test(\SimpleXMLElement $element, $value, $group = null, Registry
 	{
 		$filterInput = InputFilter::getInstance();
 
-		if (preg_match('#[<>"\'%;()&\\\\]|\\.\\./#', $value) || strlen(utf8_decode($value)) < 2
+		if (preg_match('#[<>"\'%;()&\\\\]|\\.\\./#', $value) || \strlen(utf8_decode($value)) < 2
```
Comment (HLeithner) [inline installation/src/Form/Rule/UsernameRule.php:42]:
```suggestion
		if (\preg_match('#[<>"\'%;()&\\\\]|\\.\\./#', $value) || \strlen(utf8_decode($value)) < 2
```

Comment (PhilETaylor) [reply installation/src/Form/Rule/UsernameRule.php:42]: Can you explain why? `preg_match` is not opcode optimised in PHP source code https://github.com/php/php-src/blob/9dc947522186766db4a7e2d603703a2250797577/Zend/zend_compile.c#L4192

File: installation/src/Controller/InstallationController.php (hunk 1 above)
Comment (PhilETaylor) [reply installation/src/Controller/InstallationController.php:154]: Can you explain why? `is_file` is not opcode optimised in PHP source code https://github.com/php/php-src/blob/9dc947522186766db4a7e2d603703a2250797577/Zend/zend_compile.c#L4192

**Comment by PhilETaylor** [comment]
Closing as [no one has the balls](https://github.com/joomla/joomla-cms/pull/33357#issuecomment-827627256) to merge this.

**Pull request closed without merging**